src/app/products/charges/create-charge/create-charge.component.html

```diff
@@ -0,0 +1,200 @@
+                <mat-option *ngFor="let feeFrequency of chargesTemplateData.feeFrequencyOptions" [value]="feeFrequency.id">
+                  {{ feeFrequency.value }}
+                </mat-option>
+              </mat-select>
+              <mat-error *ngIf="chargeForm.controls.feeFrequency.hasError('required')">
+                Charge Frequency is <strong>required</strong>
+              </mat-error>
+            </mat-form-field>
+
+            <mat-form-field *ngIf="chargeForm.controls.chargeTimeType.value === 9  && chargeForm.controls.addFeeFrequency.value" fxFlex="48%">
+              <mat-label>Frequency Interval</mat-label>
+              <input matInput required formControlName="feeInterval">
+              <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('required')">
+                Frequency Interval is <strong>required</strong>
+              </mat-error>
+              <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('pattern')">
+                Frequency Interval <strong>must be an integer</strong>
+              </mat-error>
+            </mat-form-field>
+
+            <mat-form-field *ngIf="chargeForm.controls.chargeTimeType.value === 6 || chargeForm.controls.chargeTimeType.value === 7" fxFlex="48%">
+              <mat-label>Due Date</mat-label>
+              <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="dueDatePicker" required formControlName="feeOnMonthDay">
+              <mat-datepicker-toggle matSuffix [for]="dueDatePicker"></mat-datepicker-toggle>
+              <mat-datepicker #dueDatePicker></mat-datepicker>
+              <mat-error *ngIf="chargeForm.controls.feeOnMonthDay.hasError('required')">
+                Due Date is <strong>required</strong>
+              </mat-error>
+            </mat-form-field>
+
+            <div *ngIf="chargeForm.controls.chargeTimeType.value === 7 || chargeForm.controls.chargeTimeType.value === 11" fxFlex="48%" fxLayout="row">
+              <mat-form-field fxFlex="80%">
+                <mat-label>Repeat Every</mat-label>
+                <input fxFlex="80%" matInput required formControlName="feeInterval">
+                <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('required')">
+                  Repeat Every is <strong>required</strong>
+                </mat-error>
+                <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('min') || chargeForm.controls.feeInterval.hasError('max')">
+                  Repeat Every <strong>must be between 1 and 12 months</strong>
+                </mat-error>
+                <mat-error *ngIf="chargeForm.controls.feeInterval.hasError('pattern')">
+                  Repeat Every <strong>must be an integer</strong>
+                </mat-error>
+              </mat-form-field>
+
+              <div fxFlex="20%" class="repeat-every-label-wrapper">
+                <mat-label class="repeat-every-label mat-h4">{{ repeatEveryLabel }}</mat-label>
+              </div>
```

Comment:
Instead of this label specified separately would it be better if we specify it above instead?

Reply:
Specified it in a brackets instead.